subscription/src/main/java/org/killbill/billing/subscription/api/user/DefaultSubscriptionBillingEvent.java

```diff
@@ -79,12 +76,24 @@ public Integer getBcdLocal() {
     public String toString() {
         return "DefaultSubscriptionBillingEvent{" +
                "type=" + type +
-               ", planName='" + planName + '\'' +
-               ", planPhaseName='" + planPhaseName + '\'' +
+               ", plan='" + plan.getName() + '\'' +
```

Comment:
Do we need a `null` check here (e.g. for cancellations)?

Reply:
From construction, `plan` and `planPhase` should never be null -- in case of cancel we take the  prev. transition `plan`